intfsyncd/intfsync.cpp

```diff
@@ -58,7 +69,9 @@ void IntfSync::onMsg(int nlmsg_type, struct nl_object *obj)
     std::vector<FieldValueTuple> fvVector;
     FieldValueTuple f("family", family);
     FieldValueTuple s("scope", scope);
+    FieldValueTuple m("if_mtu", to_string(if_mtu));
```

Comment:
can we use mtu instead of if_mtu?

Reply:
The if_mtu is defined in the following schema, please me know if it is OK to update the schema for this
https://github.com/Azure/sonic-swss/blob/d1fdf399e1e0e2e0d71fad1bce2ea7babee2eb48/doc/swss-schema.md

Reply:
I prefer to use "mtu" in this context it is "ip_mtu", but we do not need to prepend ip before the mtu.